core/src/main/java/dk/alexandra/fresco/framework/builder/numeric/AdvancedNumeric.java

```diff
@@ -109,47 +158,56 @@
   /**
    * @param input input
    * @param shifts Number of shifts
-   * @return result input >> shifts
-   */
-  DRes<SInt> rightShift(DRes<SInt> input, int shifts);
-
-  /**
-   * @param input input
-   * @param shifts Number of shifts
-   * @return result: input >> shifts<br>
+   * @return A deferred result computing <br>
+   *         result: input >> shifts<br>
    *         remainder: The <code>shifts</code> least significant bits of the input with the least
    *         significant having index 0.
    */
   DRes<RightShiftResult> rightShiftWithRemainder(DRes<SInt> input, int shifts);
 
+  /**
+   * Computes the bit length of the input.
+   * 
+   * @param input The number to know the bit length of
+   * @param maxBitLength The maximum bit length this number can have (if unknown, set this to the
+   *        modulus bit size)
+   * @return A deferred result computing the bit length of the input number.
+   */
   DRes<SInt> bitLength(DRes<SInt> input, int maxBitLength);
 
+  /**
+   * Compute the inverse of x within the field of operation
```

Comment:
I am so sorry, but checkstyle requires a period in the first sentence of a javadoc.
"First sentence of Javadoc is incomplete (period is missing) or not present."
You should make sure that this branch does not have javadoc violations - at lest on the changed/updated files